transactions/privateForwarder/create_private_forwarder.cdc

```diff
@@ -0,0 +1,19 @@
+import FungibleToken from 0xFUNGIBLETOKENADDRESS
+import ExampleToken from 0xTOKENADDRESS
+import PrivateReceiverForwarder from 0xPRIVATEFORWARDINGADDRESS
+
+transaction(receiver: Address) {
+
+    prepare(acct: AuthAccount) {
+        let recipient = getAccount(receiver)
+            .getCapability<&{FungibleToken.Receiver}>(/public/exampleTokenReceiver)
```

Comment:
I'm not sure I understand this part.

The idea is the recipient (of the FLOW) should only be able to receive FLOW from the private receiver. Which means it does not have any other `FungibleToken.Receiver`s. It only has a Vault in storage and a balance...

Maybe I'm reading something wrong.